### ISSUE TITLE
Is AppScale dead? 

### ISSUE THREAD

**Issue by cyberquarks** [AppScale/appscale#3230]
Is AppScale for Google App Engine already dead? Is it still supported or maintained?

**Comment by obino** [comment]
At this time it is not any longer supported or maintained officially by AppScale Inc.

**Comment by cyberquarks** [comment]
@obino is the last version stable and can be used for production? I suppose there are already documented guides that can be used by developers also?

**Comment by obino** [comment]
Yes the last version has been used for production up to the time we supported it. Of course you may have issue about compatibility with the current version of Google App Engine as time passes.

**Comment by cyberquarks** [comment]
Hi @obino which version of GAE SDK has been tested for production of GTS? 

ps. I assume that GTS with the supported SDK will run smoothly for years? Would that be a good assumption?
